packages/terra-form-select/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+Fixed
```

Comment:
There's been a few minor changes in the formatting of the change log entries:

https://github.com/cerner/terra-core/pull/3106

Reply:
Updated to new format